start-kafka.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+docker compose up -d
+
+sleep 5 # wait until broker is ready to accept requests
+
+echo "Creating kafka topics..."
+docker exec -it streams-app_tools_1 ./create-topics.sh
```

Comment:
I'm running Ubuntu 20.04 with the following Docker versions:
Docker version 20.10.12, build e91ed57
Docker Compose version v2.2.3

It seems `streams-app_tools_1` is Docker compose versio dependent name. My version of Docker compose names the container with hyphens instead of underscores `streams-app-tools-1`.

I'd suggest that the container name could be excplicitly set in the compose file to make it more version independent.
```
container_name: streams-app_tools_1
```

Reply:
good point thanks, I am running on Mac OS with different versions as well (docker: `20.10.6`, compose: `1.29.1`)